### ISSUE TITLE
.env fil

### ISSUE THREAD

**Issue by pchauhan77** [eduardo3g/twitter#1]
Hi,
can you share .env file and i want to integrate aws rds how can i do it? do you have any example?

**Comment by eduardo3g** [comment]
Hi!

First of all, you need to deploy the whole stack using the command `sls deploy`.

Once the deployment is done, run the command `npm run exportEnv`. It will automatically generate the .env file for you using the serverless-export-env plugin.

Cheers!

**Comment by pchauhan77** [comment]
![Capture37](https://user-images.githubusercontent.com/12031880/134462902-c763587c-b251-4bb4-bf1e-c9d7dd479806.PNG)
I am getting this error.

**Comment by eduardo3g** [comment]
Hey @pchauhan77, this error comes from the `serverless-layers` plugin. I raised a new issue #2  to keep track of it separately. I plan to open a PR today to fix it.
